README.markdown

```diff
@@ -112,6 +112,16 @@ The Hypothesis LMS app is written for python 3 and uses Node.js and `yarn` for m
     export H_CLIENT_SECRET="eVJ4***rXkk"
     export H_AUTHORITY="lms.hypothes.is"
     export H_API_URL="http://localhost:5000/api"
+
+    # An optional space-separated list of the consumer keys of the application
+    # instances for which the "auto provisioning" features should be enabled.
+    # If not given defaults to [] (i.e. the features are disabled for everyone).
```

Comment:
This sentence is a tad hard to parse without punctuation. Perhaps something like:

"Defaults to an empty list (`[]`) if none provided (i.e. the features will be disabled for all application instances)."

Reply:
Done!